Firmware/ControladorRobo/control.h

```diff
@@ -14,10 +14,10 @@
 #define umin -1.0
 #define umax 1.0
 
-//Modelo Esquerdo
+// Right model
```

Comment:
Right model is kind of vague. This is actually a linear aproximation of the system motor + wheel using a "first-order plus deadtime"  (FOPDT) transfer function, where t1 is the time constant, teta is the delay, and kp is the gain.